### PR TITLE
Enable Ruby 2.3 in Travis, make it default suite

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,24 +20,25 @@ branches:
 
 matrix:
   include:
-  - rvm: 2.1
-  - rvm: 2.0
-  - rvm: 2.2
+  - rvm: 2.2.5
+  - rvm: 2.1.9
+  - rvm: 2.0.0-p648
```
Comment (chris-rock) [inline .travis.yml]: +1

File: appveyor.yml [hunk 2]
```diff
@@ -16,10 +16,10 @@ environment:
     - ruby_version: "21"
```
Comment (chris-rock) [inline appveyor.yml]: lets add a local test for 23 as well.

Comment (jkeiser) [inline appveyor.yml]: @chris-rock not sure what you mean? The matrix in the new world:

```
    - ruby_version: "21"
      train_target: winrm://test_user@localhost:5985

    - ruby_version: "23"
      train_target: winrm://test_user@localhost:5985

    - ruby_version: "23"
      train_ssl: true
      train_target: winrm://test_user@localhost:5986
```

Comment (jkeiser) [inline appveyor.yml]: i.e. I'm not sure what a local run is--I just mindlessly replaced 22 with 23. Or did you mean add a local run for 22 as well?

**Comment by chris-rock** [comment]
Thanks @jkeiser for this improvement

**Pull request merged**